src/Scriban.Tests/TestFiles/020-interpolation/020-interpolation.txt

```diff
@@ -0,0 +1,7 @@
+{{ $"Scriban {4 + 1}.8 can interpolate strings now" }}
```

Comment:
Add more tests with:
- Multiple string interpolation in from a same string (e.g `$"Hello {a} {b} {c}"`) and other variations (interpolation starting from the beginning e.g `$"{a}"`)
- Add escape handling for `{` (e.g `$"This is not \{an interpolation}"`)